framework/core/js/package.json

```
@@ -14,7 +14,7 @@
         "focus-trap": "^6.7.1",
         "jquery": "^3.6.0",
         "jquery.hotkeys": "^0.1.0",
-        "mithril": "^2.0.4",
+        "mithril": "2.0",
```

Could we use this instead? I think it's clearer at a glance.


```suggestion
        "mithril": "~2.0.4",
```

---

oh 🙈